docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,168 @@
+
+You can also use other servers, such as Fabric, Spigot, Paper, etc.
+
+.. tip::
+
+    Configure and troubleshoot your server **BEFORE** you introduce MCDR. Otherwise, you can't distinguish whether a problem is caused by MCDR or your server
+
+    MCDR was born as a daemon for common servers of Minecraft: Java Edition. Most plugins are predicated on that. If you want to run MCDR with a Bedrock Edition server, consider using a Java Edition server with `GeyserMC <https://geysermc.org/>`__ to provide Bedrock Edition compatibility
+
+    Of course, if you don't need to make use of the plugin ecosystem designed for Java Edition servers, you can run any programs you want with MCDR
+
+Initialize
+----------
+
+Let's say you are going to start MCDR in a folder named ``my_mcdr_server``. Then you can run the following commands to initialize the environment for MCDR:
+
+.. code-block:: bash
+
+    cd my_mcdr_server
+    mcdreforged init
+
+MCDR will generate its default structure like this:
+
+.. code-block::
+
+    my_mcdr_server/
+     ├─ config/
+     ├─ logs/
+     │   └─ MCDR.log
+     ├─ plugins/
+     ├─ server/
+     ├─ config.yml
+     └─ permission.yml
+
+Animated demo:
+
+.. asciinema:: resources/init.cast
+    :rows: 10
+
+|
+
+Configure
+---------
+
+Server
+~~~~~~
+
+Remember the server you prepared? Put it into the ``server`` folder. The directory structure should be something like this:
+
+.. code-block:: diff
+
+        my_mcdr_server/
+        ├─ config/
+        ├─ logs/
+        │   └─ MCDR.log
+        ├─ plugins/
+        ├─ server/
+    ++  │   ├─ ...
+    ++  │   ├─ minecraft_server.jar
+    ++  │   └─ server.properties
+        ├─ config.yml
+        └─ permission.yml
+
+Config File
+~~~~~~~~~~~
+
+Then, edit the ``config.yml`` file to configure MCDR. You can find more information about this in :doc:`/configuration`
+
+For most users, there are 4 parts of the configuration that you need to be aware of
+
+- :ref:`configuration:language`: the language using in MCDR
+- :ref:`configuration:start_command`: how MCDR starts your server
+- :ref:`configuration:handler`: how MCDR reads your server outputs
+- :ref:`configuration:encoding, decoding`: how MCDR handles the server IO stream
+
+Read each section carefully and make sure you fill in the correct values.
+
+As a simple example, these are the ones you need to be aware of for a vanilla Minecraft 1.21 server with Java 21:
+
+.. code-block:: yaml
+
+    start_command: java -Dfile.encoding=UTF-8 -Dstdout.encoding=UTF-8 -Dstderr.encoding=UTF-8 -Xms1G -Xmx2G -jar minecraft_server.jar nogui
+
+    handler: vanilla_handler
+
+    encoding: utf8
+    decoding: utf8
+
+If you're confusing about why this command is much longer than we mentioned earlier, read the **encoding, decoding** section again
+
+RCON
+~~~~
+
+Optionally, you may enable the RCON protocol to make some specific plugins work properly or more efficiently. Check the details in :ref:`configuration:rcon`
```

Comment:
tbh, I should really move the RCON feature into a plugin, instead of inside MCDR. Maybe that's one of the MCDR 3.0 TODO items

as for the present, just remove this RCON section to avoid making user thinks about that MCDR might requires RCON To work